client/components/notice/index.jsx

```diff
@@ -109,10 +124,14 @@ export class Notice extends Component {
 			'is-dismissable': showDismiss,
 		} );
 
+		const iconName = icon || this.getIcon();
+		const iconNeedsDrop = GRIDICONS_WITH_DROP.includes( iconName );
+
 		return (
 			<div className={ classes }>
 				<span className="notice__icon-wrapper">
-					<Gridicon className="notice__icon" icon={ icon || this.getIcon() } size={ 24 } />
+					{ iconNeedsDrop && <span class="notice__icon-wrapper-drop" /> }
```

Comment:
There may be some positioning issues here. This addition is creating an additional circle [as noted by](https://github.com/Automattic/wp-calypso/pull/35949#issuecomment-528074972) @sixhours:

![Screen Shot 2019-09-05 at 10 07 23](https://user-images.githubusercontent.com/841763/64323784-6ca88080-cfc5-11e9-8b7f-08837af5384e.png)

Reply:
@sixhours That’s on me. I’ll fix it right away. Thanks for reporting.